lab_3_generate_by_ngrams/main.py

```diff
@@ -76,6 +107,15 @@ def get_token(self, element_id: int) -> Optional[str]:
 
         In case of corrupt input arguments or arguments not included in storage, None is returned
         """
+        if (
+            not isinstance(element_id, int) or
+            len(self._storage) <= element_id
+        ):
+            return None
+        token = [i for i in self._storage if self._storage[i] == element_id][0]
```

Comment:
looks like there must be just one such element, so using list and indexing is a bit extra here

Reply:
rewritten by filter()